R/pr.R

```diff
@@ -499,21 +499,24 @@ pr_run <- function(pr,
 #' Add a static route to the `plumber` object
 #'
 #' @template param_pr
-#' @param path The mounted path location of the static folder
 #' @param direc The local folder to be served statically
+#' @param path The mounted path location of the static folder
 #'
 #' @examples
 #' \dontrun{
 #' pr() %>%
-#'   pr_static("/path", "./my_folder/location") %>%
+#'   pr_static("./my_folder/location", "/path") %>%
 #'   pr_run()
 #' }
 #'
 #' @export
+# TODO: Validate if this makes sense?
+# Arguments path and direc are in reverse order compared to plumber annotation `@assets direc {path}`.
+# Default path is `/public` for annotation `@assets`
```

Comment:
What the comment says

Reply:
I think `@assets`'s order is wrong. I'd like to keep the original `pr_static()` setup.  

It might be nice to change the docs to not use `@assets` (due to the confusing order) and instead use....

```r
#' @plumber
function(pr) {
  pr %>%
    pr_static(path, direc)
}
```

Reply:
My hypothesis is that the initial `@asset` args order was this way because `path` was optional. It default to `/public` when not provided. I could change the order and still making `path` optional by assuming:

One argument on line, set as directory
#* @asset directory
Two arguments on line, set as path and directory
#* @asset path directory

Is it ok?

Reply:
I will assume this is ok and go ahead with the changes.

Reply:
Also, what a sweeping job you did. Good job.

Reply:
Unfortunately, we can not change the behavior of the tag `@assets` tag. This would definitely be breaking behavior. 

If anything we'd have to add a new tag (ex: `@static) and I'd like to pass on that for now.

When we add support for httpuv static paths (which don't support filters), we can make sure the new tag names are good.

Reply:
Let's keep them in their original order then, even if mismatched.